build.gradle

```diff
@@ -98,13 +95,13 @@ subprojects {
         withSourcesJar()
     }
 
+    //need to decide who owns the testing infrastructure
+    //random junit versions picked below.
     test {
         useJUnitPlatform()
         dependencies {
-            testCompile("org.junit.jupiter:junit-jupiter-api:5.7.0")
```

Comment:
ideally we should put any explicit versions into the 'constraints' section in the top level build.gradle.